awscli/customizations/s3/subcommands.py

```diff
@@ -725,7 +725,7 @@ class CpCommand(S3TransferCommand):
             "or <S3Uri> <S3Uri>"
     ARG_TABLE = [{'name': 'paths', 'nargs': 2, 'positional_arg': True,
                   'synopsis': USAGE}] + TRANSFER_ARGS + \
-                [METADATA, METADATA_DIRECTIVE, EXPECTED_SIZE, RECURSIVE]
+                [METADATA, METADATA_DIRECTIVE, EXPECTED_SIZE, RECURSIVE, REQUEST_PAYER]
```

Comment:
For the `cp` and `sync`, it would be great if this can be added to the `TRANSFER_ARGS` list so it will get included in the `mv` command as well and it does not have to be explicitly added to each list for those commands.